pages/portfolios/index.vue

```diff
@@ -0,0 +1,8 @@
+<template>
+	<main>
+		<h1>
+			Portfolios.
+		</h1>
+		<p style="color:lightslategray">Beautiful, minimalistic, free</p>
```

Comment:
Inline styles mixed with scss?

Reply:
Copied from the original html, has now been fixed.